### ISSUE TITLE
About pretrain time

### ISSUE THREAD

**Issue by 1787648106** [fudan-zvg/SeaFormer#21]
Thank you very much for your excellent work. May I ask, what is your experimental hardware configuration? For example, the model of the GPU; In addition, how long did you complete the pre training on imagenet1k?

**Comment by wwqq** [comment]
Hi @1787648106 , V100*8. Tiny:~1.5 days;  Small:~1.8 days;  Base:~2.2 days Large:~2.5days

**Comment by 1787648106** [comment]
thx for your response ~

**Comment by 1787648106** [comment]
> Hi @1787648106 , V100*8. Tiny:~1.5 days; Small:~1.8 days; Base:~2.2 days Large:~2.5days

@wwqq hello, the pretrain launch command, should set the -b(--batch-size) is 16 on a node with 8 gpus or set 128 in the launch command? thanks a lot.

**Comment by 1787648106** [comment]
> Hi @1787648106 , V100*8. Tiny:~1.5 days; Small:~1.8 days; Base:~2.2 days Large:~2.5days

hello~I pretrained the model on ImageNet1k, but the training time kept about ~5 days. A800 * 8, base model version, amp on. Are there some accelerating skills used ? Thanks again

**Comment by 1787648106** [comment]
> > Hi @1787648106 , V100*8. Tiny:~1.5 days; Small:~1.8 days; Base:~2.2 days Large:~2.5days
> 
> hello~I pretrained the model on ImageNet1k, but the training time kept about ~5 days. A800 * 8, base model version, amp on. Are there some accelerating skills used ? Thanks again

@wwqq hello, I'm so sorry to bother you again. About the accelerating in pretrain.

**Comment by wwqq** [comment]
@1787648106 Hello, try setting the batch size and learning rate to twice their original values.
